_episodes_rmd/13-supp-data-structures.Rmd

```diff
@@ -260,23 +267,26 @@ nchar("Software Carpentry")
 
 In R matrices are an extension of the numeric or character vectors. They are not
 a separate type of object but simply an atomic vector with dimensions; the
-number of rows and columns.
+number of rows and columns. As with atomic vectors, the elements of a matrix must
+be of the same data type.
```

Comment:
Since we are adding content here anyway, how about turning it into a challenge? Like "Which data types can a matrix contain?" => A solution would be "Any, as long as they are the same", right?

Reply:
@captainalan: What do you think about this as a `{: .challenge}` block?

Reply:
A challenge block looks good here; it's a step or two of reasoning from the info presented in this lesson

Reply:
@captainalan: If you want this to be in the lesson release (see #415), please create this challenge before next Tuesday.

Reply:
@katrinleinweber I have took up the challenge of creating a challenge 😎
https://github.com/swcarpentry/r-novice-inflammation/pull/433

Reply:
Updating an existing PR is possible by pushing more commits to that branch ;-)